### PR TITLE
Add RUN button to the main screen

### PR DESCRIPTION
**Pull request by baruchiro**
image

This PR is not a completed work.
I mostly changed the folder name, and the `App` component name, and add the RUN button.

We will adjust the log view (the `result`) and the config in the next work.

### CODE REVIEW

File: src/components/App/MainContent.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    btnColor() {
+      return colors[this.succeeded];
+    }
+  },
+  methods: {
+    async scrape() {
+      this.scraping = true;
+      scrapeAndUpdateOutputVendors()
+        .then((results) => {
+          this.results = results;
```
Comment (baruchiro) [inline src/components/App/MainContent.vue:51]: The `results` object/string/value is just for now

File: src/components/App/MainContent.vue [hunk 2]
```diff
@@ -49,9 +49,11 @@ export default {
...
         .then((results) => {
           this.results = results;
+          this.succeeded = true;
         })
         .catch((error) => {
           this.results = error.message;
+          this.succeeded = false;
         })
         .finally(() => {
           console.log(this.results);
```
Comment (brafdlog) [inline src/components/App/MainContent.vue:59]: lint

**Comment by baruchiro** [comment]
I added more changes- I converted the `store` to TS and removed all old modules (`states`)

File: src/store/modules/Config.ts [hunk 3]
```diff
@@ -1,36 +1,38 @@
...
 export const GET_EXPORTER_GETTER = 'GET_EXPORTER_GETTER';
 
-const state = {};
+const state: Config = defaultConfig;
 
-const mutations = {
-  addImporter: (state, importer) => state.scraping.accountsToScrape.push(importer),
-  removeImporter: (state, importerId) => {
+const mutations = <MutationTree<Config>>{
+  addImporter: (state: Config, importer) => state.scraping.accountsToScrape.push(importer),
```
Comment (brafdlog) [inline src/store/modules/Config.ts:15]:
```suggestion
  addImporter: (state: Config, importer: AccountToScrapeConfig) => state.scraping.accountsToScrape.push(importer),
```

File: src/store/modules/Config.ts [hunk 4]
```diff
@@ -1,36 +1,38 @@
...
+  removeImporter: (state: Config, importerId) => {
```
Comment (brafdlog) [inline src/store/modules/Config.ts:16]:
```suggestion
  removeImporter: (state: Config, importerId: string) => {
```

**Comment by brafdlog** [comment]
@baruchiro Since its already shabat, I fixed the comments and will merge. If you disagree with any of the changes tell me and I will fix.

File: src/store/modules/Config.ts (hunk 4 above)
Comment (baruchiro) [reply src/store/modules/Config.ts]: Is this fix because of any error, or just for use types everywhere we can?

Comment (brafdlog) [reply src/store/modules/Config.ts]: Just to add the type since we know what it is

Comment (baruchiro) [reply src/store/modules/Config.ts]: OK, I will do it from now on

**Pull request merged**